nixos/default.nix

```
@@ -44,6 +44,7 @@ exposeModules ./. [
   ./hardware/hetzner-online/amd.nix
   ./hardware/hetzner-online/intel.nix
   ./hardware/hetzner-online/ex101.nix
+  ./hardware/hetzner-online/rx170.nix
```

I would generalize this, since we got two models that are basically the same SOC with different hardware builtin:

```suggestion
  ./hardware/hetzner-online/arm.nix
```